editoast/README.md

```diff
@@ -46,6 +46,7 @@ cargo test -- --test-threads=4
 Here a list of components to help you in your development (see CI jobs if necessary):
 
 - [rustfmt](https://github.com/rust-lang/rustfmt): Format the whole code `cargo fmt`
+- [cargo-tomlfmt](https://github.com/tbrand/cargo-tomlfmt): Format the `Cargo.toml` files with `cargo tomlfmt --path path/to/Cargo.toml`
```

Comment:
```suggestion
```

Reply:
Ooops, thanks for catching that. Fixed in [b9100c4](https://github.com/osrd-project/osrd/pull/7053/commits/b9100c473857a09778a67c789a60e9c6c5bb1a40).